VimeoNetworking/Sources/VimeoClient.swift

```diff
@@ -233,56 +231,56 @@ final public class VimeoClient
                 }
             }
             
-            if request.cacheFetchPolicy == .CacheOnly
+            if request.cacheFetchPolicy == .cacheOnly
             {
                 return RequestToken(path: request.path, task: nil)
             }
             
-        case .NetworkOnly, .TryNetworkThenCache:
+        case .networkOnly, .tryNetworkThenCache:
             break
         }
         
-        let success: (NSURLSessionDataTask, AnyObject?) -> Void = { (task, responseObject) in
-            dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0)) {
+        let success: (URLSessionDataTask, Any?) -> Void = { (task, responseObject) in
+            
+            DispatchQueue.global(qos: .userInitiated).async {
```

Comment:
This is ok? Just want to double check. Mike mentioned something about the converter doing weird stuff with this.